autoload/coqtail.vim

```diff
@@ -429,36 +442,50 @@ let s:cmd_opts = {
   \ 'CoqGotoGoalPrev': '-bar -bang',
   \ 'CoqToggleDebug': '-bar'
 \}
-function! s:cmddef(name, act) abort
-  " Start Coqtail first if needed
-  let l:act = a:name !~# '\(Start\|Stop\|Interrupt\)$'
-    \ ? printf('if s:running() || coqtail#start() | %s | endif', a:act)
-    \ : a:act
+
+" a:aux indicates whether we're defining the command
+" for an auxiliary buffer or the main buffer
+function! s:cmddef(aux, name, act) abort
+  if a:aux
+    let l:act =
+          \ printf( 'let _tmp_panel = ' .
+          \             'coqtail#panels#switch(g:coqtail#panels#main) | ' .
+          \         'if _tmp_panel != g:coqtail#panels#none | ' .
+          \           '%s | ' .
+          \           'call coqtail#panels#switch(_tmp_panel) | ' .
+          \         'endif',
+          \ a:act)
```

Comment:
This shouldn't be necessary anymore right?

Reply:
Ah you're right. I reverted it in my latest commit.